shared/params/testnet_prater_config.go

```diff
@@ -0,0 +1,37 @@
+package params
+
+import (
+	eth1Params "github.com/ethereum/go-ethereum/params"
+)
+
+// UsePraterNetworkConfig uses the Pyrmont specific
+// network config.
+func UsePraterNetworkConfig() {
+	cfg := BeaconNetworkConfig().Copy()
+	cfg.ContractDeploymentBlock = 4367322
+	cfg.BootstrapNodes = []string{
+		// TODO(8612): Define bootstrap nodes.
+	}
+	OverrideBeaconNetworkConfig(cfg)
+}
+
+// UsePraterConfig sets the main beacon chain
+// config for Pyrmont.
+func UsePraterConfig() {
+	beaconConfig = PraterConfig()
+}
+
+// PraterConfig defines the config for the
+// Prater testnet.
+func PraterConfig() *BeaconChainConfig {
+	cfg := MainnetConfig().Copy()
+	cfg.MinGenesisTime = 1616508000
```

Comment:
```suggestion
	cfg.MinGenesisTime = 1614588812
```
https://github.com/eth2-clients/eth2-testnets/blob/master/shared/prater/config.yaml#L22

It is `1614588812`, you are using genesis_time + genesis_delay

Reply:
We don't add these together here. The min genesis time should be min genesis time.

Reply:
Actually, what happened was I copied the value from the PR description. 

![Screenshot from 2021-03-16 15-49-07](https://user-images.githubusercontent.com/7246818/111377922-3a261c00-866f-11eb-8f7d-b8623c469b46.png)

This genesis time is not the min genesis time and you were correct. Thanks!